torch/csrc/jit/export.cpp

```diff
@@ -181,7 +181,28 @@ void encodeGraph(onnx::GraphProto * p_g, const std::shared_ptr<Graph> & g, const
   JIT_ASSERT(p_g != nullptr);
   p_g->set_name("torch-jit-export");
 
+  // The jit tracer may record some unnecessary information.
+  // We should remove the unused input and initializer.
+  // Gather all the names of useful nodes and filter out unused ones.
```

Comment:
I think this comment needs to be elaborated, because we had a pretty long discussion about what is going on here, and it needs to be clear that this code is responsible for eliminating the correspondence between `state_dict` and the parameters of an exported ONNX model, so basically you are SOL if you want to manually interpret the parameters.